benchmarks/build.gradle

```diff
@@ -77,7 +77,7 @@ tasks.named("run").configure {
   executable = "${BuildParams.runtimeJavaHome}/bin/java"
   args << "-Dplugins.dir=${buildDir}/plugins" << "-Dtests.index=${buildDir}/index"
   dependsOn "copyExpression", "copyPainless"
-  systemProperty 'java.library.path', file("../libs/native/libraries/build/platform/${platformName()}-${os.arch}")
+  systemProperty 'es.nativelibs.path', file("../libs/native/libraries/build/platform/${platformName()}-${os.arch}")
```

Comment:
IIRC this was causing issues on Linux systems that have both x64 and x86 (i386/32 bit) libraries installed - since we are not messing anymore with java.library.path, do you think this is going to fix that?
(If not, I don't think you need to address it in this PR - it is an existing issue and this will not make the situation worse)

Reply:
The issue with x64 and x86 was when looking for libsystemd in the system paths. For Elasticsearch we only ship a single platform dir with each distribution. So I don't think we should encounter that issue, and plus we don't look for libsystemd anymore.